modules/apps/27-interchain-accounts/host/keeper/keeper_test.go

```diff
@@ -256,3 +262,20 @@ func (suite *KeeperTestSuite) TestParams() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestUnsetParams() {
+	// this state should not be possible outside of tests
+	chainID := "testchain"
+	header := tmproto.Header{
+		ChainID: chainID,
+		Height:  1,
+		Time:    suite.coordinator.CurrentTime.UTC(),
+	}
+
+	app := simapp.NewSimApp(log.NewNopLogger(), dbm.NewMemDB(), nil, true, map[int64]bool{}, simapp.DefaultNodeHome, 5, simapp.MakeTestEncodingConfig(), simtestutil.EmptyAppOptions{}, baseapp.SetChainID(chainID))
+	ctx := app.GetBaseApp().NewContext(true, header)
```

Comment:
another approach could be to delete the key:
```go
store := suite.chainA.GetContext().KVStore(suite.chainA.GetSimApp().GetKey(hosttypes.SubModuleName))
store.Delete(icatypes.ParamsKey)

```

Reply:
is there a need to create a new app in this test case when everything should be bootstrapped by `suite.SetupTest()`?

Reply:
done